import org.junit.Test;
import static org.junit.Assert.assertEquals; 
import org.junit.runner.RunWith;
import org.junit.runners.Suite;
import org.junit.runners.Suite.SuiteClasses;

import frontEnd.Calculator;

	public class CalculatorTest { 
		private int nberrors=0;
		@Test 
		public void testAdd(){
			Calculator calculator = new Calculator();
			double result = calculator.add(10,50);
			if (result != 60) {
				throw new IllegalStateException ("Bad Resut"+ result); 
		  
		} 
		}

		public static void main(String[] args) {  
			CalculatorTest test = new CalculatorTest();
			try {
				test.testAdd();
			}catch (Throwable e) {
				test.nberrors++;
				e.printStackTrace();
			}
			if (test.nberrors>0) {
				throw new IllegalStateException("There were" + test.nberrors
						+ "error(s)");
			}
		}
  
			} 
	


